### PR TITLE
Add quiet flag for checkov input

### PR DESCRIPTION
**Pull request by Dineshk77**
# Description:
- Add quiet flag for checkov input
- This is to avoid dumping large output and just display only the failure information
- Also large output ends up in Argument list too long errors.

### CODE REVIEW

File: .github/workflows/terraform.yaml [hunk 1]
```diff
@@ -17,6 +17,11 @@ on:
         type: string
         default: ""
         required: false
+      checkov_output_format:
+        description: Checkov output format
+        type: string
+        default: "sarif"
+        required: false
```
Comment (lawliet89) [inline .github/workflows/terraform.yaml:24]: @Dineshk77 is SARIF readable by humans? We don't have SARIF upload enabled for all repos (cost reason).

**Review by niroz89 — COMMENTED** [review]
@Dineshk77 bro, isn't this is handled by `quiet: true` parameter ? we want it to be `sarif` so we can use GHAS + CodeQL

**Comment by Dineshk77** [comment]
> @Dineshk77 bro, isn't this is handled by `quiet: true` parameter ? we want it to be `sarif` so we can use GHAS + CodeQL

Ah okay in that case I think we need to truncate the output (not sure will it lose data, if so we won't have full data), the problem here is with the output which results in `Argument list too long` when `output content` is large.

**Comment by Dineshk77** [comment]
> > @Dineshk77 bro, isn't this is handled by `quiet: true` parameter ? we want it to be `sarif` so we can use GHAS + CodeQL
> 
> Ah okay in that case I think we need to truncate the output (not sure will it lose data, if so we won't have full data), the problem here is with the output which results in `Argument list too long` when `output content` is large.

@niroz89 My bad, I thought `quiet` flag is true by default. Seems its not. Reducing the output display by setting the `quiet` flag `true` resolves the `Argument list too long`  issue

**Review by lawliet89 — APPROVED** [review]

**Comment by lawliet89** [comment]
Maybe rename the PR @Dineshk77

**Review by niroz89 — APPROVED** [review]

**Pull request merged**